forest/daemon/src/daemon.rs

```diff
@@ -437,6 +440,14 @@ fn chain_path(config: &Config) -> PathBuf {
     PathBuf::from(&config.client.data_dir).join(&config.chain.name)
 }
 
+fn get_actual_network_name(internal_network_name: &str) -> &str {
```

Comment:
In Forest to avoid ambiguities, we use _network_ for everything that is network related. See network section of config:
```
[network]
listening_multiaddr = "/ip4/0.0.0.0/tcp/0"
bootstrap_peers = []
mdns = false
kademlia = true
target_peer_count = 75
```

And _chain_ is more for "network" in the sense of Filecoin, mainnet, calibnet, devnet, etc.
So I would rename `get_actual_network_name` to `get_actual_chain_name`.

Reply:
corrected